integration_tests/run_pyspark_from_build.sh

```diff
@@ -40,18 +40,20 @@ else
     # support alternate local jars NOT building from the source code
     if [ -d "$LOCAL_JAR_PATH" ]; then
         CUDF_JARS=$(echo "$LOCAL_JAR_PATH"/cudf-*.jar)
+        AVRO_JARS=$(echo "$LOCAL_JAR_PATH"/spark-avro*.jar)
```

Comment:
Have we thought about how to test that we can still run things without avro on the classpath? I see that you thought about this and put in code to dynamically load things, which is really great work. I just want to be sure that I don't accidentally break it in the future. Especially because there is no separate maven package that isolates all code that touches avro from code that should not touch it. spark-avro is just on the classpath so I could accidentally include something I shouldn't in the wrong location and now we are in trouble.

Reply:
Yeah, Great comments. Currently, the premerge CI can cover part of this by running `mvn tests` or `mvn -P spark311tests test`. For the long term, I think we should add a pipeline for premerge to skip avro tests and spark-avro jar and run all tests.

I will file a follow up on this